modules/local/calculate_basequality/main.nf

```diff
@@ -0,0 +1,41 @@
+process CALCULATE_BASEQUALITY {
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://community-cr-prod.seqera.io/docker/registry/v2/blobs/sha256/ba/ba5dc444f779fd6c19aaf6b5e767e441f4578b3d8841280a5b6e6a3a23a05044/data' :
+        'ccommunity.wave.seqera.io/library/pip_pysam:0756c7708ed1598b' }"
+
+    input:
+    tuple val(meta), path(input)
+
+    output:
+    tuple val(meta), path("*.json") , emit: json
+    path('versions.yml')            , emit: versions
+
+    script:
+    """
+    #!/usr/bin/env bash
+    if [ -f "${meta.fastp_json}" ]; then
+        mv "${meta.fastp_json}" "${meta.id}.json"
```

Comment:
Nextflow should be making a symlink here, right? So this would not move the actual file that exists outside the workdir, right?

Reply:
I think it moves the copy (in the working dir) meaning original file is not affected. Nextflow never apply changes to the input files, unless you define output dir to the same directory.

Reply:
You were right, I just tested this. It deletes the original file 🗡️